src/main/java/net/neoforged/moddevgradle/dsl/RunModel.java

```diff
@@ -140,7 +143,16 @@ public void jvmArgument(String arg) {
      *
      * @see ModModel
      */
-    public abstract SetProperty<ModModel> getMods();
+    public abstract SetProperty<ModModel> getLoadedMods();
+
+    /**
+     * @deprecated This method conflicts with {@link NeoForgeExtension#getMods()}. Use {@link #getLoadedMods()} instead.
+     */
+    @Deprecated(forRemoval = true)
+    public SetProperty<ModModel> getMods() {
+        logger.error("RunModel#getMods() is deprecated and will be removed in ModDevGradle 3. Use RunModel#getLoadedMods() instead.");
```

Comment:
Error so that it prints a red message in the console. A warning would only print a white message.